app/models/bookmark.rb

```diff
@@ -0,0 +1,7 @@
+class Bookmark < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :playspace
+
+  validates :user_id, presence: true
+  validates :playspace_id, presence: true, uniqueness: true
```

Comment:
am I supposed to add this to bookmarks_helper or controller... or somewhere else?